### PR TITLE
EUI-7483 Linked hearings displayed twice when clicking on change option

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-7483

### Change description ###
Hearings : Linked hearings displayed twice when clicking on change option

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/hearings/containers/request-hearing/hearing-link/hearing-link.component.ts [hunk 1]
```diff
@@ -48,25 +49,30 @@ export class HearingLinkComponent extends RequestHearingPageFlow implements OnIn
...
+          });
         }
-      );
-    }, () => {
-      this.router.navigate(['/hearings/error']);
+        this.showSpinner = false;
+      },
+      error: () => {
+        this.router.navigate(['/hearings/error']);
+      }
```
Comment (phillip-whitaker-hmcts) [inline src/hearings/containers/request-hearing/hearing-link/hearing-link.component.ts:75]: can you just remove this unnecessary wrapping in brackets - 

`error: () => this.router.navigate(['/hearings/error'])`

Comment (johnbenjamin-hmcts) [reply src/hearings/containers/request-hearing/hearing-link/hearing-link.component.ts]: I have done the changes as you suggested in this PR and in PR 2638.

**Review by phillip-whitaker-hmcts — APPROVED** [review]

File: src/hearings/containers/request-hearing/hearing-link/hearing-link.component.ts [hunk 2]
```diff
@@ -3,9 +3,11 @@ import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { ActivatedRoute, Router } from '@angular/router';
 import { select, Store } from '@ngrx/store';
 import { Subscription } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
+import { HearingLinksStateData } from 'src/hearings/models/hearingLinksStateData.model';
```
Comment (Daniel-Lam) [inline src/hearings/containers/request-hearing/hearing-link/hearing-link.component.ts:7]: @johnbenjamin-hmcts - please change this import to a relative path

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@johnbenjamin-hmcts - just a minor change to an import required; the rest is OK

**Review by Daniel-Lam — APPROVED** [review]

File: src/hearings/containers/request-hearing/hearing-link/hearing-link.component.ts (hunk 2 above)
Comment (johnbenjamin-hmcts) [reply src/hearings/containers/request-hearing/hearing-link/hearing-link.component.ts]: I am not sure how it sneaks in like spyware even after being extra cautious :(

Thank you @Daniel-Lam I have now changed the path.

**Pull request merged**